Session_3/closure.js

```diff
@@ -0,0 +1,14 @@
+//Closures and Scopes
+// http://www.codewars.com/kata/closures-and-scopes/train/javascript
+
+function createFunctions(n) {
+    var callbacks = [];
+
+    for (let i = 0; i < n; i++) {
```

Comment:
Rename the variables. single word vars are very bad practice

Reply:
In codewars default code content was like :
function createFunctions(n) {
  var callbacks = [];

  for (var i=0; i<n; i++) {
    callbacks.push(function() {
      return i;
    });
  }
  return callbacks;
}

Only I changed from var to let to show proper behavior as expected in test cases.